binder/postBuild

```diff
@@ -3,3 +3,8 @@ python -m bash_kernel.install
 
 # Trust our notebook
 jupyter trust demo/World\ population.ipynb
+
+# Set up extensions for JupyterLab and Notebook
+jupyter labextension install jupyterlab-jupytext
```

Comment:
That's correct. If we don't include that line, then we get asked if the jupytext extension for jupyterlab should be built, so it's better to do that in advance